x-pack/plugins/lens/public/app_plugin/csv_download_provider/csv_download_provider.tsx

```diff
@@ -143,28 +141,23 @@ export const downloadCsvShareProvider = ({
 
     return [
       {
-        ...menuItemMetadata,
+        shareMenuItem: {
+          name: panelTitle,
+          icon: 'document',
+          disabled: !csvEnabled,
+          sortOrder: 1,
+        },
         label: 'CSV' as const,
         reportType: 'lens_csv' as const,
         generateExport: downloadCSVHandler,
+        warnings: getWarnings(activeData),
         ...(atLeastGold()
           ? {
-              helpText: (
-                <FormattedMessage
-                  id="xpack.lens.share.helpText"
-                  defaultMessage="Export a CSV of this visualization."
-                />
-              ),
-              generateExportButton: (
-                <FormattedMessage id="xpack.lens.share.export" defaultMessage="Export file" />
-              ),
+              disabled: !csvEnabled,
               renderLayoutOptionSwitch: false,
               getJobParams: undefined,
-              showRadios: true,
             }
           : {
-              isDisabled: !csvEnabled,
```

Comment:
This was not being used nor is it a property on `ShareMenuItemV2`. Additionally we do not support these radios being disabled, `csvEnabled` is currently all or nothing (i.e. hide or show the `Export` tab).